platform/p4/docker-sonic-p4/orchagent.sh

```diff
@@ -14,7 +14,7 @@ ORCHAGENT_ARGS="-d /var/log/swss "
 ORCHAGENT_ARGS+="-b 8192 "
 
 # Set synchronous mode if it is enabled in CONFIG_DB
-SYNC_MODE=$(sonic-cfggen -d -v DEVICE_METADATA.localhost.synchronous_mode)
+SYNC_MODE=$(sonic-cfggen -d -t /etc/sonic/synchronous_mode.j2 | jq -r '.synchronous_mode')
```

Comment:
>sonic-cfggen [](start = 12, length = 12)

sonic-cfggen is slow in the warm/fast reboot critical timeline. So we optimize them aggressively.
It is possible to limit only one call in each script to generate everything into a big variable, and extract each config from the variable.

Reply:
Reduced `sonic-cfggen` usage.